index.bs

```diff
@@ -2581,6 +2581,38 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
```

Comment:
If I read this correctly, I think this is the wrong place.  This new section should come just before 1.5.1 AudioNode Creation. Or maybe after.

Why?  Because in all other nodes the Interface section ends with the IDL.  If you do that here, I think the IDL ends up being in the lifetime section.

Reply:
Fixed.